cmd/osbuild-worker/jobimpl-depsolve.go

```diff
@@ -42,11 +43,10 @@ func (impl *DepsolveJobImpl) Run(job worker.Job) error {
 	if err != nil {
 		switch err.(type) {
 		case *rpmmd.DNFError:
-			result.ErrorType = worker.DepsolveErrorType
+			result.JobError = clienterrors.WorkerClientError(clienterrors.ErrorDepsolveJob, err.Error())
```

Comment:
So the DepsolveError is a DNFError, see https://dnf.readthedocs.io/en/latest/api_exceptions.html.

So ErrorDNFDepsolveError seems more apprioriate here? This is the user input failure we don't need to pay attention to.

Reply:
see https://github.com/osbuild/osbuild-composer/pull/2192#discussion_r786789901